### ISSUE TITLE
App stops when opening multi-volume rar file 

### ISSUE THREAD

**Issue by HuiiBuh** [TeamAmaze/AmazeFileManager#2094]
**Describe the bug**
App crashes. 

**To Reproduce**
I just tried to open an ear archive which was split into multiple parts. 

**Expected behavior**
App does not stop, but extract the files. 

**Smartphone (please complete the following information):**
 - Poco F1
 - OS: Android 10
 - Rooted: Yes
 - Version: 3.5.0

**Additional context**
Stack trace: https://del.dog/sadylicrag

**Comment by VishalNehra** [comment]
@HuiiBuh I don't think we support ear files. Is that a typo? As for the crash. It's a pretty serious one. How do you end up opening a file that doesn't have an extension using Amaze's internal archive viewer?

**Comment by HuiiBuh** [comment]
Oh yes that was a typo.
I meant rar files.
No this is the default behavior

**Comment by VishalNehra** [comment]
@HuiiBuh Ok, so your file doesn't have an extension right? Can you attach the file?

**Comment by HuiiBuh** [comment]
No the file has an extension.
It is a `rar` file used for compressing files (like a `zip` file) .
I am afraid that I can't share the file with you. But it looks to me as if this error happens every time you open a `rar` archive which is split in multiple files

**Comment by VishalNehra** [comment]
@TranceLove can you please check if a quick fix is possible, else will move to 3.6

**Comment by TranceLove** [comment]
> @TranceLove can you please check if a quick fix is possible, else will move to 3.6

[junrar](https://github.com/junrar/junrar), now at 7.4.0, supports multipart volumes as well as password-protected rars. I also tried updating the lib with the one we're using (0.7!), seems a drop-in replacement which compiles without error.

However, I see there will be some extra handling needed, either in code or being smart (like user can still extract from part 1 of the rar even when they tapped part 2). Adding tests for regression (that I saw some `Seekable` `Channel` came across in their code that may cause incompatibility in lower SDKs), not going to think it's feasible for time window of this short.
